modules/platforms/dotnet/Apache.Ignite.Core/Impl/Cache/Query/QueryCursorFieldMetadataImpl.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Core.Impl.Cache.Query
+{
+    using System;
+    using Apache.Ignite.Core.Binary;
+    using Apache.Ignite.Core.Cache.Query;
+    using Apache.Ignite.Core.Impl.Binary;
+
+    /// <summary>
+    /// Query cursor field metadata implementation.
+    /// </summary>
+    public class QueryCursorFieldMetadataImpl : IQueryCursorFieldMetadata
```

Comment:
Please remove `Impl` suffix. It is present on some types because of Java naming legacy, but it is against Microsoft standards, so we try to avoid this.

Reply:
**Important**: make this class `internal`